michelson/configuration.md

```diff
@@ -335,8 +344,7 @@ of Michelson code. We list these configuration cells here:
                   <returncode exit=""> 1 </returncode>
     ```
 
-
-6. The following cell is a debugging aid, indicating whether an `#Assume`
+7. The following cell is a debugging aid, indicating whether an `#Assume`
```

Comment:
Lets move this to bullet points in a future PR?

Reply:
Sounds good.